requirements/ginkgo.in

```diff
@@ -0,0 +1,8 @@
+Django<2.0
```

Comment:
You can pin the right Django version for Ginkgo (django==1.8.18), since the vulnerable version are >= 1.11.0, < 1.11.22. @andrey-canon